PolyEngine/Games/SandboxRendering/Src/RenderingSandbox.cpp

```diff
@@ -33,48 +37,56 @@ void RenderingSandbox::Init()
-	Poly::TransformComponent* lightTrans = world->GetComponent<Poly::TransformComponent>(PointLight);
-	dirLightTrans->SetLocalTranslation(PointLightPos);
-
-	auto Dummy = DeferredTaskSystem::SpawnEntityImmediate(world);
-	DeferredTaskSystem::AddComponentImmediate<Poly::TransformComponent>(world, Dummy);
-	DeferredTaskSystem::AddComponentImmediate<Poly::MeshRenderingComponent>(world, Dummy, "Models/bullet/bullet.obj", eResourceSource::GAME);
-	Poly::TransformComponent* dummyTrans = world->GetComponent<Poly::TransformComponent>(Dummy);
-	dummyTrans->SetLocalScale(Vector(0.5f, 0.5f, -1.0f));
-	dummyTrans->SetLocalTranslation(DirLightPos);
-	dummyTrans->SetLocalRotation(DirLightRot);
-
+	DeferredTaskSystem::AddComponentImmediate<Poly::PointLightComponent>(world, PointLight, Color(0.0f, 1.0f, 1.0f), 2.0f, 0.1f);
+	
+	AddPointLights();
+
+	// Dummy = DeferredTaskSystem::SpawnEntityImmediate(world);
+	// DeferredTaskSystem::AddComponentImmediate<Poly::TransformComponent>(world, Dummy);
+	// DeferredTaskSystem::AddComponentImmediate<Poly::MeshRenderingComponent>(world, Dummy, "Models/bullet/bullet.obj", eResourceSource::GAME);
+	// Poly::TransformComponent* dummyTrans = world->GetComponent<Poly::TransformComponent>(Dummy);
+	// dummyTrans->SetLocalScale(Vector(0.5f, 0.5f, -1.0f));
+	// dummyTrans->SetLocalTranslation(PointLightPos);	
+	
 	float yPos = (float)Engine->GetRenderingDevice()->GetScreenSize().Height;
 	auto textDispaly = DeferredTaskSystem::SpawnEntityImmediate(world);
 	DeferredTaskSystem::AddComponentImmediate<Poly::ScreenSpaceTextComponent>(world, textDispaly, Vector{ 0.0f, yPos ,0.0f }, "Fonts/Raleway/Raleway-Heavy.ttf", eResourceSource::ENGINE, 32, "Kill count: 0");
 
-	auto player = DeferredTaskSystem::SpawnEntityImmediate(world);
-	DeferredTaskSystem::AddComponentImmediate<Poly::TransformComponent>(world, player);
-	DeferredTaskSystem::AddComponentImmediate<Poly::MeshRenderingComponent>(world, player, "Models/tank2/bradle.3ds", eResourceSource::GAME);
-	Poly::TransformComponent* playerTransform = world->GetComponent<Poly::TransformComponent>(player);
-	playerTransform->SetLocalTranslation(Vector(-3.0f, 0.0f, 5.0f));
-	playerTransform->SetLocalScale(10.0f);
-	playerTransform->SetLocalRotation(Quaternion(Vector::UNIT_Y, -90_deg) * Quaternion(Vector::UNIT_X, -90_deg));
+	// auto player = DeferredTaskSystem::SpawnEntityImmediate(world);
+	// DeferredTaskSystem::AddComponentImmediate<Poly::TransformComponent>(world, player);
+	// DeferredTaskSystem::AddComponentImmediate<Poly::MeshRenderingComponent>(world, player, "Models/tank2/bradle.3ds", eResourceSource::GAME);
+	// Poly::TransformComponent* playerTransform = world->GetComponent<Poly::TransformComponent>(player);
+	// playerTransform->SetLocalTranslation(Vector(-2.0f, 0.0f, 0.0f));
+	// playerTransform->SetLocalScale(10.0f);
+	// playerTransform->SetLocalRotation(Quaternion(Vector::UNIT_Y, -90_deg) * Quaternion(Vector::UNIT_X, -90_deg));
```

Comment:
Remove commented code. You can always bring it back using git.

Reply:
Done, sorry for that